standard/classes.md

```diff
@@ -5354,7 +5354,9 @@ Task types can vary in their exact definition, but from the language’s point o
 >
 > *end example*
 
-A task builder type is a class or struct type that corresponds to a specific task type ([§15.15.2](classes.md#15152-task-type-builder-pattern)). The task builder type must have the same declared accessibility as its corresponding task type.
+A task builder type is a class or struct type that corresponds to a specific task type ([§15.15.2](classes.md#15152-task-type-builder-pattern)). The task builder type must exactly match the declared accessibility as its corresponding task type.
```

Comment:
For some reason I can't leave a suggestion here, but I think "of its" would read better than "as its". (Or change to "have the same declared accessibility as its corresponding task type").